release/models/aft/openconfig-aft-common.yang

```diff
@@ -379,6 +414,29 @@ submodule openconfig-aft-common {
     }
   }
 
+  grouping aft-common-entry-nexthop-encap-outer-header-state {
+    description
+      "Outer header fields of encapsulation applied on a next-hop";
+
+    leaf ttl {
+      type uint32;
+      description
+        "This leaf reflects the configured TTL value that is used in the outer
+         header during packet encapsulation. When this leaf is not set, the TTL
+         value of the inner packet is copied over as the outer packet's TTL
+         value during encapsulation.";
+    }
+
+    leaf tos {
+      type uint16;
+      description
+        "This leaf reflects the configured TOS value that is used in the outer
+         header during packet encapsulation. When this leaf is not set, the TOS
```

Comment:
Depending on the implementation there can be 3 ways:
1. Setting the value of tos by configuration.
2. Copy from inner payload.
3. Setting default tos.
The proposed changes are catering to 1 and 2 of the above ways, please accommodate all possible ways.

Reply:
IIUC your question: This path is for telemetry only. So we are not setting any values, its the value that is going to be used. So if we had a default value configured [or cause nothing is configured], then this will be a non-zero value that will be streamed, so in that case, 1 and 3 are the same here from a telemetry prespective.

Let me know if I misunderstood.

Reply:
To be more precise please update the description as below:
 "This leaf reflects the configured TOS value" as "This leaf reflects the configured/default TOS value"

also if default TOS value is zero, zero will be streamed out.
is this the expectation?

Reply:
Updated documentation. 

> also if default TOS value is zero, zero will be streamed out.
is this the expectation?
Yes.